styles/styles.css

```diff
@@ -0,0 +1,168 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+body {
+  font-family: 'Noto Sans', sans-serif;
+  margin: 5px;
+}
+/* 
+#heading {
+  display: flex;
+  flex-wrap: nowrap;
+} */
+
+#startrly {
+  margin-right: 400px;
+  margin-left: 1px;
+}
+
+#try-now {
+  margin-left: 400px;
+  margin-right: 300px;
+}
+
+nav ul {
+  list-style-type: none;
```

Comment:
I think you could do this in one step for all the list items in the whole document by doing this once (which will help dry out your code)

li {
list-style-type: none;
}